docs/collection-tables.rst

```diff
@@ -0,0 +1,89 @@
+.. _collection-tables:
+
+=================
+Collection Tables
+=================
+
+Imagine you are modelling gene expression data from a given collection.
+Ideally, you would want all expression values organized in a table where
+rows represents samples and columns represent genes. Class
+``CollectionTables`` gives you just that (and more).
+
+We will present the functionality of ``CollectionTables`` through an
+example. We will:
+
+- Create an instance of ``CollectionTables`` and examine it's attributes
+- Fetch raw expression counts and only keep `TIS genes`_ with sufficient
+  expression
+- Compute ``log2(TPM + 1)`` of expression values and visualize samples
+  in a simple PCA plot
+
+.. _`TIS genes`: https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-019-2100-3
+
+First, connect to a Resolwe server, pick a collection and create
+and instance of ``CollectionTables``::
+
+    import resdk
+    res = resdk.Resolwe(url='https://app.genialis.com/')
```

Comment:
Note: we should have `app.genialis.com` by default.

Reply:
Current default is `localhost`. Are we sure about app being the default url?

Reply:
@dblenkus, what's your take on that?